airflow/providers/http/hooks/http.py

```diff
@@ -38,6 +38,14 @@
     from airflow.models import Connection
 
 
+def _url_from_endpoint(base_url: str | None, endpoint: str | None) -> str:
+    """Combine base url with endpoint."""
+    print(f"base_url: {base_url}, endpoint: {endpoint}")
```

Comment:
If we're logging stuffs, we might want to use `import logging` instead

Reply:
sorry debugging print left on code, fixed!